tests/tests.py

```diff
@@ -275,6 +275,16 @@ def test_save(self):
         dirs, files = self.backend.listdir('')
         self.assertEqual(files.count(name), 1)
 
+    @patch('tests.utils.FakeSwift.objects', new=deepcopy(CONTAINER_CONTENTS))
+    def test_save_gzip(self):
```

Comment:
Maybe you could mock/patch GzipFile for this test and check if it gets called instead.
Doesn't this test now just check if the file gets added and not if it gets gzipped?